config-model/src/test/java/com/yahoo/vespa/model/container/xml/IdentityBuilderTest.java

```diff
@@ -19,16 +23,50 @@
     @Test
     public void identity_config_produced() throws IOException, SAXException {
         Element clusterElem = DomBuilderTest.parse(
-                "<jdisc id='default' version='1.0'>",
-                "  <identity>",
-                "    <domain>domain</domain>",
-                "    <service>service</service>",
-                "  </identity>",
-                "</jdisc>");
-
-        createModel(root, clusterElem);
+                "<jdisc id='default' version='1.0'/>");
+        String deploymentXml = "<deployment version='1.0' athenz-domain='domain' athenz-service='service'>\n" +
+                               "    <test/>\n" +
+                               "    <prod>\n" +
+                               "        <region active='true'>default</region>\n" +
+                               "    </prod>\n" +
+                               "</deployment>\n";
+
+        ApplicationPackage applicationPackage = new MockApplicationPackage.Builder()
+                .withDeploymentSpec(deploymentXml)
+                .build();
+
+
+        // Override root
+        root = new MockRoot("root", applicationPackage);
+        createModel(root, DeployState.createTestState(applicationPackage), clusterElem);
+
         IdentityConfig identityConfig = root.getConfig(IdentityConfig.class, "default/component/" + Identity.CLASS);
         assertEquals("domain", identityConfig.domain());
         assertEquals("service", identityConfig.service());
     }
+
+    @Test
+    public void athenz_service_override() throws IOException, SAXException {
```

Comment:
Again, test should have a name that is a requirement, e.g. athenz_service_can_be_overridded_for_...